meta-facebook/wc-mb/src/platform/plat_sensor_table.c

```diff
@@ -254,6 +254,11 @@ bool pal_is_time_to_poll(uint8_t sensor_num, int poll_time)
 	return true;
 }
 
+uint8_t get_hsc_pwr_reading(int *reading)
+{
+	return get_sensor_reading(SENSOR_NUM_PWR_HSCIN, reading, GET_FROM_CACHE);
+}
+
```

Comment:
Check to make sure reading isn't NULL using macro recently added.

Reply:
Normally, There should have a pointer check for input parameters, but in this case, function "get_hsc_pwr_reading()" is just a transporter of "get_sensor_reading()". It already has a certain return value for invalid pointer case. Should I add another return value or just using SENSOR_UNSPECIFIED_ERROR which is the same as "get_sensor_reading()" invalid pointer case, thanks.

Reply:
I would add another check here. That way if the function were to change in the future the check won't be forgotten.

Reply:
Understood.
New commit has been updated, please have a look thanks.